README.md

```diff
@@ -1,6 +1,6 @@
 # JHipster Online
 
-[![Azure DevOps Build Status][azure-devops-image]][azure-devops-url-main]  [![Build Status][travis-image]][travis-url]  [![Docker Pulls](https://img.shields.io/docker/pulls/jhipster/jhipster-online.svg)](https://hub.docker.com/r/jhipster/jhipster-online/)
+[![Azure DevOps Build Status][azure-devops-image]][azure-devops-url-main] [![Build Status][travis-image]][travis-url] [![Docker Pulls](https://img.shields.io/docker/pulls/jhipster/jhipster-online.svg)](https://hub.docker.com/r/jhipster/jhipster-online/)
```

Comment:
Since Azure pipelines are now removed we need to remove this badge. 
```suggestion
[![Build Status][travis-image]][travis-url] [![Docker Pulls](https://img.shields.io/docker/pulls/jhipster/jhipster-online.svg)](https://hub.docker.com/r/jhipster/jhipster-online/)
```

Reply:
Done at https://github.com/jhipster/jhipster-online/pull/168